cowprotocol/lookup_tables/valid_reward_targets_5492689.sql

```diff
@@ -0,0 +1,43 @@
+--noqa: disable=all
+select distinct cowRewardTarget--, 'ethereum' as blockchain
+from cow_protocol_ethereum.VouchRegister_evt_Vouch
+inner join query_4056263 -- valid bonding pools
+    on pool_address = bondingPool
+    and sender = creator
+
+union distinct
+select distinct cowRewardTarget--, 'gnosis' as blockchain
+from cow_protocol_gnosis.VouchRegister_evt_Vouch
+inner join query_4056263 -- valid bonding pools
+    on pool_address = bondingPool
+    and sender = creator
+
+union distinct
+select distinct cowRewardTarget--, 'base' as blockchain
+from cow_protocol_base.VouchRegister_evt_Vouch
+inner join query_4056263 -- valid bonding pools
+    on pool_address = bondingPool
+    and sender = creator
+
+union distinct
+select distinct cowRewardTarget--, 'arbitrum' as blockchain
+from cow_protocol_arbitrum.VouchRegister_evt_Vouch
+inner join query_4056263 -- valid bonding pools
+    on pool_address = bondingPool
+    and sender = creator
+
+union distinct
+select distinct cowRewardTarget--, 'avalanche_c' as blockchain
+from cow_protocol_avalanche_c.VouchRegister_evt_Vouch
+inner join query_4056263 -- valid bonding pools
+    on pool_address = bondingPool
+    and sender = creator
+
+/*
+union distinct
```

Comment:
due to issues with namespaces for Polygon (which we should ping the Dune team about), as a workaround we have this query that collects all vouching events for all networks

https://dune.com/queries/5513654